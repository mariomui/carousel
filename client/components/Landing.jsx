/* eslint-disable no-useless-constructor */
/* eslint-disable class-methods-use-this */
import React from 'react';

class Landing extends React.Component {
  constructor() {
    super();
    this.state = {
      image_url: [],
    };
  }

  render() {
    return (
      <div> bye bye </div>
    );
  }
}

export default Landing;
