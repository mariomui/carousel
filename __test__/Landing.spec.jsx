import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Landing from '../client/components/Landing';

Enzyme.configure({ adapter: new Adapter() });

test('this tests landing to match snapshot', () => {
  const tree = shallow(<Landing />);
  expect(toJson(tree)).toMatchSnapshot();
});
