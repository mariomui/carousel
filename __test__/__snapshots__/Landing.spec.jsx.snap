// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this tests landing to match snapshot 1`] = `
<div>
   this iddsnt working! 
</div>
`;
